endpoints/getting-started/client/main.go

```diff
@@ -16,37 +16,32 @@
 package main
 
 import (
-	"crypto/rsa"
-	"crypto/x509"
-	"encoding/pem"
-	"errors"
+	"context"
 	"flag"
 	"fmt"
-	"io/ioutil"
+	"io"
 	"log"
 	"net/http"
 	"os"
 	"time"
 
-	"golang.org/x/oauth2/google"
-	"golang.org/x/oauth2/jws"
+	"cloud.google.com/go/auth/credentials"
 )
 
 var (
-	url                 = flag.String("url", "", "The request url. Required.")
-	audience            = flag.String("audience", "", "The audience for the JWT. equired")
-	serviceAccountFile  = flag.String("service-account-file", "", "Path to service account JSON file. Required.")
-	serviceAccountEmail = flag.String("service-account-email", "", "Path email associated with the service account. Required.")
+	url                = flag.String("url", "", "The request url. Required.")
+	audience           = flag.String("audience", "", "The audience for the JWT. equired")
+	serviceAccountFile = flag.String("service-account-file", "", "Path to service account JSON file. Required.")
 )
 
 func main() {
 	flag.Parse()
 
-	if *audience == "" || *url == "" || *serviceAccountFile == "" || *serviceAccountEmail == "" {
+	if *audience == "" || *url == "" || *serviceAccountFile == "" {
 		fmt.Println("requires: --url, --audience, --service-account-file, --service-account-email")
```

Comment:
I suspect you might have wanted to drop , --service-account-email" from this Println based on the removal of the serviceAccountEmail